src/OpenTelemetry.Api/Correlations/CorrelationContextEntry.cs

```diff
@@ -16,35 +16,35 @@
 
 using OpenTelemetry.Internal;
 
-namespace OpenTelemetry.Context
```

Comment:
would it make sense to keep the namespace Context? it is still the right thing. I am imagining in the future we can have one more type of context objects and would make sense to keep all of them under one namespace.

Reply:
I kept the Context namespace.